Test.Utility/TestEnvironmentBuilder.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Castle.Core;
+using Messerli.Test.Utility;
+
+namespace Messerli.Test.Utility
+{
+    public class TestEnvironmentBuilder
+    {
+        private readonly List<TestFile> _testFiles = new List<TestFile>();
+
+        public TestEnvironmentBuilder AddTestFile(TestFile file)
+        {
+            _testFiles.Add(file);
+            return this;
```

Comment:
Make the builder functional by returning a new `TestEnvironmentBuilder`

Reply:
Addressed in https://github.com/messerli-informatik-ag/test-utility/pull/11/commits/c8d6ab46905a494bd91a564b0e4a71f90a2ae690